### DISCUSSION TITLE
How to remove padding/margin highlight on mouse hover?

### DISCUSSION THREAD

**Discussion by Agobin** [GrapesJS/grapesjs#3709] [Q&A] [answered]
If a component has padding or margin and it's not selected, hovering the mouse over the component highlights the padding and margin with background colors.

How can I prevent this highlighting effect? I've tried setting highlightable to false but it doesn't work.
Thanks for your help.

**Comment by artf** [comment, marked as answer]
Hi @Agobin check these options
https://github.com/artf/grapesjs/blob/61b00cb3b9bd7bf989ab2a1d562160ed9e5e3748/src/editor/config/config.js#L17-L21

**Reply by feriwnarta** [reply]
Hello @artf, I've been looking at the use of GrapeJS in studio.grapes.js. I'd like to ask how to make a padding bar and margin bar appear when a component is clicked, marked with the class <div class="flex absolute gjs-cv-unscale left-0" style="top: 50%; translate: 0px -50%; transform-origin: left center;"><div class="bg-blue-450 text-white border-2 border-white !box-content bg-[#7fc65e] h-[20px] w-[3px] mx-2 cursor-ew-resize rounded-full text-sm"></div></div> in the canvas spot section.
